apps/backend/discovery-server/src/main/resources/application.yaml

```
@@ -0,0 +1,17 @@
+eureka:
+  instance:
+    hostname: discovery-server
+  client:
+    register-with-eureka: false
+    fetch-registry: false
+# actuator
+management:
+  endpoints:
+    web:
+      exposure:
+        include: health,info
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)